docs/dev/internals/storage.md

```diff
@@ -2,6 +2,8 @@
 layout: docu
 title: Storage
 expanded: Development
+redirect_from:
```

Comment:
This file shouldn't be moved - there should only be one copy that contains the information for all the versions at once